abc.templates/base-workflows/contents/guardian-admin.yml

```diff
@@ -133,4 +133,4 @@ jobs:
         run: |-
           guardian run \
           -dir="${DIRECTORY}" \
-          -- ${TERRAFORM_COMMAND}
+          -- ${TERRAFORM_COMMAND//\"}
```

Comment:
What is going on here? This looks like you are adding a quote, not dropping.

Reply:
In bash, you can use pattern matching to substitute characters with the format `${parameter//pattern/string}`.

From `man bash`, 
> If there are two slashes separating _parameter_ and _pattern_, all matches of _pattern_ are replaced with _string_. If _string_ is null, matches of pattern are deleted and the ‘/’ following _pattern_ may be omitted.

So in this case, I'm matching all double quotes in the `TERRAFORM_COMMAND` string variable and removing it.

Reply:
TIL.

Might make sense to add a comment somewhere

Reply:
This is going to remove _all_ quotes, which could be unintended if my `TERRAFORM_COMMAND` is something like `terraform "foo bar"` where I intend "foo bar" to be a single argument. I think you want something like https://stackoverflow.com/questions/9733338/shell-script-remove-first-and-last-quote-from-a-variable

Reply:
Fair argument, but that also doesn't fix the initial problem.

It turns out, saving `inputs.command` into an environment variable `$TERRAFORM_COMMAND` messed with the way the arguments are parsed (I'm assuming it uses the default `IFS` delimiter to parse the variable).  
For example, an input of `plan "foo bar"` will parse to `["plan", "\"foo", "bar\""]`

But using the `inputs.command` directly in GitHub Actions expression syntax produced the desired result.